docker-compose-dev.yml

```diff
@@ -70,6 +70,25 @@ services:
       driver: "json-file"
       options:
         max-size: 1M
+  worker-exclusive:
+    extends:
+      file: docker-compose-common.yml
+      service: conversionbase
+    volumes:
+      - ./osmaxx:/home/py/osmaxx
+      - ./web_frontend:/home/py/web_frontend
+      - ./osmaxx_conversion_service:/home/py/osmaxx_conversion_service
+    build:
+      context: .
+      dockerfile: Dockerfile.worker
+    command: [honcho, -f, ./osmaxx_conversion_service/Procfile.worker, start]
+    environment:
+      - DJANGO_SECRET_KEY=insecure!4
+      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.local
```

Comment:
no, since this file is overwriting and adding the docker-compose.yml file when used as specified (`docker-compose -f docker-compose.yml -f docker-compose-dev.yml`)